presentation/src/main/java/com/keeply/presentation/ui/folder/UncategorizedDisplay.kt

```diff
@@ -26,7 +26,10 @@ import com.keeply.presentation.core.theme.KeeplyTheme
 import com.keeply.presentation.ui.folder.component.UncategorizedCardItem
 
 @Composable
-fun UncategorizedDisplay() {
+fun UncategorizedDisplay(
+    images: kotlinx.collections.immutable.ImmutableList<com.keeply.domain.folder.model.FolderImage> = kotlinx.collections.immutable.persistentListOf(),
```

Comment:
p3: 요것만 import로 수정 부탁드림당 
```suggestion
import com.keeply.domain.folder.model.FolderImage
import kotlinx.collections.immutable.ImmutableList
import kotlinx.collections.immutable.persistentListOf
    
            images: ImmutableList<FolderImage> = persistentListOf(),
```